extensions/shared/external-media/editor.scss

```diff
@@ -358,6 +358,9 @@ $grid-size: 8px;
 }
 
 // Reset placeholder button margin.
-.components-placeholder__fieldset .components-dropdown .components-button {
-	margin-right: 8px;
+.components-placeholder__fieldset,
```

Comment:
What is this used for? It doesn't look like this is namespaced to the extension.

The rule below can also be simplified down to 
```css
.editor-post-featured-image .components-dropdown .jetpack-external-media-browse-button {
    margin-right: 8px;
}
```

Reply:
It's used to add button padding in the featured image panel and the media placeholder. It resolves to:
```css
.components-placeholder__fieldset .components-dropdown .jetpack-external-media-browse-button,
.editor-post-featured-image .components-dropdown .jetpack-external-media-browse-button {
	margin-right: 8px;
}
```

See #16059

Reply:
Weird so it does, is this SASS shorthand?

<img width="974" alt="Screen Shot 2020-06-11 at 1 40 37 PM" src="https://user-images.githubusercontent.com/1270189/84437007-339edb80-abe9-11ea-9071-fe0a4d9dba46.png">

I'd prefer if we could avoid it, as I'd read this as apply to 
```css
.components-placeholder__fieldset,
.editor-post-featured-image .components-dropdown .jetpack-external-media-browse-button {
    margin-right: 8px;
}
````